go/private/rules/test.bzl

```diff
@@ -112,6 +112,9 @@ def _go_test_impl(ctx):
         "l_test=" + external_source.library.importpath,
     )
     arguments.add("-pkgname", internal_source.library.importpath)
+
+    # Boolean Go flags require =, so we can't just use the attr as an arg value, we need to convert it ourselves.
+    arguments.add("-register_timeout_handler={}".format("true" if ctx.attr.register_timeout_handler else "false"))
```

Comment:
This allocates a new string for each target. Instead, you could just have an `if ... else`. Alternatively, since the flag has a default, only adding the flag when it's not the default would be an option as well.

Reply:
Done